X/xkbcommon/build_tarballs.jl

```diff
@@ -3,36 +3,28 @@
 using BinaryBuilder
 
 name = "xkbcommon"
-version = v"0.9.1"
+version = v"1.4.1"
 
 # Collection of sources required to build xkbcommon
 sources = [
     ArchiveSource("https://xkbcommon.org/download/libxkbcommon-$(version).tar.xz",
-                  "d4c6aabf0a5c1fc616f8a6a65c8a818c03773b9a87da9fbc434da5acd1199be0"),
-    DirectorySource("./bundled"),
+                  "943c07a1e2198026d8102b17270a1f406e4d3d6bbc4ae105b9e1b82d7d136b39"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/libxkbcommon-*/
-
-# We need to run `wayland-scanner` on the host system
 apk add wayland-dev
-
-atomic_patch -p1 ../patches/meson_build.patch
-atomic_patch -p1 ../patches/meson_options.patch
-
 mkdir build && cd build
 meson .. --cross-file="${MESON_TARGET_TOOLCHAIN}" \
-    -Denable-docs=false \
-    -Dnative-wayland-scanner="/usr/bin/wayland-scanner"
+    -Denable-docs=false
 ninja -j${nproc}
 ninja install
 """
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = [p for p in supported_platforms() if Sys.islinux(p)]
+platforms = filter!(p -> arch(p) != "armv6l" && Sys.islinux(p), supported_platforms())
```

Comment:
What's the problem with `armv6l` here?

Reply:
From what I remember, we don't have the dependencies on those either.